### PR TITLE
Segresnet algo update

### PR DESCRIPTION
**Pull request by myron**
bug fixes, and performance improvements

### CODE REVIEW

File: auto3dseg/algorithm_templates/segresnet2d/scripts/utils.py [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+        },
+    }
+
+    if log_output_file is not None:
+        log_config["handlers"]["file"]["filename"] = log_output_file
+        log_config["handlers"]["file"]["level"] = "DEBUG"
+    else:
+        log_config["handlers"]["file"]["level"] = "CRITICAL"
+
+    if debug or bool(os.environ.get("SEGRESNET_DEBUG", False)):
```
Comment (wyli) [inline auto3dseg/algorithm_templates/segresnet2d/scripts/utils.py:42]: this requires import os
======================================================================
ERROR: test_autorunner (__main__.TestAutoRunner)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/research-contributions/research-contributions/core/tests/test_integration_autorunner.py", line 124, in test_autorunner
    runner.run()
  File "/__w/research-contributions/research-contributions/core/monai/apps/auto3dseg/auto_runner.py", line 775, in run
    bundle_generator.generate(self.work_dir, num_fold=self.num_fold, allow_skip=self.allow_skip)
  File "/__w/research-contributions/research-contributions/core/monai/apps/auto3dseg/bundle_gen.py", line 589, in generate
    gen_algo.export_to_disk(output_folder, name, fold=f_id)
  File "/tmp/tmpcqeax5go/work_dir/algorithm_templates/segresnet2d/scripts/algo.py", line 315, in export_to_disk
  File "/__w/research-contributions/research-contributions/core/monai/apps/auto3dseg/bundle_gen.py", line 161, in export_to_disk
    self.fill_records = self.fill_template_config(self.data_stats_files, self.output_path, **kwargs)
  File "/tmp/tmpcqeax5go/work_dir/algorithm_templates/segresnet2d/scripts/algo.py", line 84, in fill_template_config
  File "/tmp/tmpcqeax5go/work_dir/algorithm_templates/segresnet2d/scripts/utils.py", line 42, in logger_configure
NameError: name 'os' is not defined

Comment (myron) [reply auto3dseg/algorithm_templates/segresnet2d/scripts/utils.py:43]: you're right. I just fixed it

**Pull request merged**